trade-finance-manager-ui/templates/utilisation-reports/_macros/premium-payments-table.njk

```diff
@@ -43,7 +45,17 @@
     {% endif %}
   {% endmacro %}
 
-  <table class="govuk-table premium-payments-table" data-module="moj-sortable-table" data-cy="premium-payments-table" id="premium-payments-table">
+  <table class="govuk-table premium-payments-table" data-module="moj-sortable-table" data-cy="premium-payments-table">
+    {% if errorMessage %}
+      {{ govukErrorMessage({
+        text: errorMessage,
+        id: "premium-payments-table-error",
+        attributes: {
+          "data-cy": "premium-payments-table--error",
+          "tabindex": "-1"
```

Comment:
What's the reason for the tab index please, I think this could cause an issue RE accessibility/screenreaders

Reply:
To allow screenreader focus to shift to the element when the link is clicked. It is needed for elements that error summaries that link to that are normally non-focusable elements e.g. divs and spans, as opposed to actual input elements which are focusable by default.

Reply:
We first discovered this after we got it back as a failure on an accessibility audit on a similar page that didn't have the tabindex set

Reply:
I see, after seeing the design. Makes sense. Thanks